sdk/cosmosdb/cosmos/samples-dev/typescript/src/AADAuth.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
At a high level, there shouldn't need to be a `typescript/src` folder in samples-dev. You can flatten the directory structure down to have only `samples-dev/AADAuth.ts` (and its siblings), and reorganize the utils like `shared` in `samples-dev/shared/handleError.ts`.